CHANGELOG.md

```diff
@@ -4,144 +4,22 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
-#### [1.2.2](https://github.com/DeMoorJasper/parcel-plugin-svelte/compare/1.2.1...1.2.2)
```

Comment:
I'm gonna get rid of the changelog, it's not that usefull imo

Reply:
Fair enough!